public/app/routes/routes.tsx

```diff
@@ -528,7 +528,7 @@ export function getAppRoutes(): RouteDescriptor[] {
 }
 
 // evaluates access control permission, using fallback if access control is disabled
-const evaluatePermission = (fallback: () => string[], action: AccessControlAction): string[] => {
+export const evaluatePermission = (fallback: () => string[], action: string): string[] => {
```

Comment:
Why would we change the type to string?

Reply:
I think it's because we'd want to use ENT or OSS AccessControlAction seamlessly

Reply:
Just a thought, would it make sense to move that to contextSrv?

Reply:
What Gabriel said :)
And I was considering moving it to `contextSrv`, but I think it'll only be used in two places. But still happy to move it if you think it'd be cleaner.